sklearn/pipeline.py

```diff
@@ -1018,7 +1022,14 @@ def make_union(*transformers, **kwargs):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+<<<<<<< HEAD
+        ..versionchanged:: v0.20
+           `n_jobs` changed from ``1`` to ``None``
+
+    verbose : boolean, optional(default=False)
+=======
     verbose : bool, default=False
+>>>>>>> upstream/master
```

Comment:
forgotten conflict...

Reply:
I guess, I've found where this conflict comes from.

In [the documentation for the version 0.22.1 ](https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.make_union.html?highlight=make_union) the options for the `verbose` parameter are written like this: 

```verbose : boolean, optional(default=False)```

and in the [dev version](https://scikit-learn.org/dev/modules/generated/sklearn.pipeline.make_union.html#sklearn.pipeline.make_union) like this:

```verbose : bool, default=False```

So I think I should combine the dev version with my code to get this in the end:

        ..versionchanged:: v0.20
           `n_jobs` changed from ``1`` to ``None``
    verbose : bool, default=False

and then commit again. Right?

Reply:
Exactly. For each conflict keep the last version available from scikit-learn master. Thanks!